android-sdk/android/src/main/res/values/logto.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="logto_redirect_scheme">logto_redirect_scheme_placeholder</string>
+    <string name="logto_redirect_scheme" />
```

Comment:
do we still need this if we can have a convention like `logto-callback://:appId/:connectorId`?